File: PooCC01CNetz/Semana02/Python/Ejercicios2/Celular.py
class Celular:
    def __init__(self):
        self.modelo = ""
        self.precio = 0.0
        self.activo = False

    def calculaIVA(self):
        iva = self.precio * 0.10
        return iva

    def descuento(self, porcentDesc):
        desc = self.precio * (porcentDesc * 0.01)
        return desc

File: PooCC01CNetz/Semana02/Java/Ejercicios2/Celular.java
public class Celular {
    // Declaro variables dentro de la clase
    public String modelo;
    public double precio;
    public boolean activo;

    // Declaro dos metodos
    public double calculaIVA() {
        double iva = precio * 0.10;
        return iva;
    }

    public double descuento(int porcenDesc) {
        double desc = precio * (porcenDesc * 0.01);
        return desc;
    }
}
